<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWMainWind</class>
 <widget class="QMainWindow" name="QWMainWind">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWMainWind</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txtEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>761</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actNew"/>
    <addaction name="actOpen"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actNew"/>
   <addaction name="actOpen"/>
   <addaction name="actCopy"/>
   <addaction name="actBold"/>
  </widget>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actNew">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actCopy">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/edit.png</normaloff>:/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actCopy</sender>
   <signal>triggered()</signal>
   <receiver>txtEdit</receiver>
   <slot>copy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>219</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
